pkg/torcx/types.go

```diff
@@ -33,6 +33,8 @@ const (
 	DefaultTagRef = "com.coreos.cl"
 	// DEFAULT_PROFILE is the default profile name used
 	DEFAULT_PROFILE_NAME = "vendor"
+	// SYSTEMD_DIR is the directoy where we'll install all systemd files
+	SYSTEMD_DIR = "/run/systemd"
```

Comment:
(duplicating OOB conversation)
For the record, `/run/systemd` is essentially a duplicate of `/usr/lib/systemd/...` but has precedence. `/run/systemd/transient` is only used for transient units created by the d-bus API or `systemd-run`. However, we might need to do things like install network files.